testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -485,15 +493,19 @@ def test_average_structure_ref_frame(self, universe):
         universe.trajectory[0]
         ref, rmsd = _get_aligned_average_positions(self.ref_files, u)
         avg = align.AverageStructure(universe, ref_frame=ref_frame).run()
-        assert_almost_equal(avg.results.universe.atoms.positions, ref,
-                            decimal=4)
-        assert_almost_equal(avg.results.rmsd, rmsd)
+        assert_allclose(avg.results.universe.atoms.positions, ref,
+                        rtol=0, atol=1.5*(1e-4),
+                        err_msg="assert_allclose failed for positions")
+        assert_allclose(avg.results.rmsd, rmsd,
+                        rtol=0, atol=1.5*(1e-4),
```

Comment:
Don't you want `1e-7` since `assert_almost_equal` is using default arguments in the original here?

Reply:
Yeah, that's a mistake. I will correct this.